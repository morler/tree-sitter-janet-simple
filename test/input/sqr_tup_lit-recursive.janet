[:html [:body [:p "surely you must be joking"]]]
