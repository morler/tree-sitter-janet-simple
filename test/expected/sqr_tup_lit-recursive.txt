(source [0, 0] - [1, 0]
  (sqr_tup_lit [0, 0] - [0, 48]
    (kwd_lit [0, 1] - [0, 6])
    (sqr_tup_lit [0, 7] - [0, 47]
      (kwd_lit [0, 8] - [0, 13])
      (sqr_tup_lit [0, 14] - [0, 46]
        (kwd_lit [0, 15] - [0, 17])
        (str_lit [0, 18] - [0, 45])))))
